config/assets.pspeu.yaml

```diff
@@ -259,11 +280,11 @@ files:
         vram: 0x09237700
         assets:
           - [0x80, skip]
-          - [0x24FF0, cmpgfx, D_pspeu_0925C670, 128, 32, 4]
-          - [0x25220, cmpgfx, D_pspeu_0925C8A0, 128, 32, 4]
-          - [0x25478, cmpgfx, D_pspeu_0925CAF8, 128, 32, 4]
-          - [0x25708, cmpgfx, D_pspeu_0925CD88, 128, 32, 4]
-          - [0x25918, cmpgfx, D_pspeu_0925CF98, 128, 32, 4]
+          - [0x24FF0, cmpgfx, stage_name_en, 128, 64, 4]
+          - [0x25220, cmpgfx, stage_name_it, 128, 64, 4]
+          - [0x25478, cmpgfx, stage_name_sp, 128, 64, 4]
+          - [0x25708, cmpgfx, stage_name_ge, 128, 64, 4]
+          - [0x25918, cmpgfx, stage_name_fr, 128, 64, 4]
```

Comment:
@gamezter I think this is still not quite right but it is slightly closer, you can see the full text at least. Still think maybe this needs tweaks in the assets tool, I tried playing around with 128x128 2bpp and some other ratios but couldn't quite work it out.